src/react-native-cool-modals/ios/RNCMPortal.m

```diff
@@ -0,0 +1,93 @@
+//
+//  Portal.m
```

Comment:
Isn't this the same file that we have under `ios/Portal.m` ? Maybe xCode copied the files when you added it to the project